ski_lint/utils.py

```diff
@@ -43,17 +45,33 @@ def get_non_ascii_files(*filenames: str) -> dict[str, Optional[str]]:
     }
 
 
-def is_ascii(s: Union[bytes, bytearray]) -> bool:
-    """Return `True` if *string* is ASCII."""
-    return chardet.detect(s)["encoding"] in ALLOWED_ENCODINGS
+def get_non_ascii_lines(filename: str) -> list[LineAnalysisResult]:
+    """Return all lines with non-ASCII chars from *filename*."""
+
+    results = []
+    with open(filename, "r") as f:
```

Comment:
```suggestion
    with open(filename) as f:
```
read mode is default

Reply:
Ok, I'll remove it.